### ISSUE TITLE
Error in plotting a graph with labels

### ISSUE THREAD

**Issue by LucaF89** [hopandfork/jgnuplot#16]
There is a problem trying to plot a graph with Labels.
The error is:
```Calling GNUPlotRunner...
Setting value at 0,0 to xxx (an instance of class java.lang.String)
Calling GNUPlotRunner...
readLine() failed
Exception in thread "Thread-0" java.lang.NullPointerException
	at org.hopandfork.jgnuplot.runtime.GNUPlotRunner.run(GNUPlotRunner.java:52)
	at java.lang.Thread.run(Thread.java:745)
java.lang.NullPointerException
	at org.hopandfork.jgnuplot.runtime.ProcessRunner.cmdExec(ProcessRunner.java:56)
	at org.hopandfork.jgnuplot.runtime.GNUPlotRunner.run(GNUPlotRunner.java:50)
	at java.lang.Thread.run(Thread.java:745)
Exception in thread "Thread-1" java.lang.NullPointerException
	at org.hopandfork.jgnuplot.runtime.GNUPlotRunner.run(GNUPlotRunner.java:52)
	at java.lang.Thread.run(Thread.java:745)
 ```

**Comment by grussorusso** [comment]
The problem is that `ProcessRunner` doesn't check for failure from the spawned process (i.e., `gnuplot`). In turn, gnuplot fails with following error:

`"work.gnuplot", line 9: undefined variable: FIRST`

Looking at the generated plot script, you can see that the error is in this line:

`set label "sssss" at FIRST 0.0,0.0`

Maybe `FIRST` is not a valid option in gnuplot (or in current version, at least).

**Comment by grussorusso** [comment]
Ok, solved. Position specifier must be lower case (e.g., `first`). Fixed by #17.

BTW, I am going to open a new issue because `ProcessRunner` has to handle failures without producing a NullPointerException!
